Automattic/jetpack
# Publicize: register `republicize` feature

This PR registers and sets the availability of the `republicize` feature. This feature has already been added to dotcom server.
#### Changes proposed in this Pull Request:

* Publicize: register republicize feature

#### Jetpack product discussion




#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test with a Jetpack site
* Open the console
* Check the `republicize` feature is available, or not, depending on the site plan. For now, this feature should be available for Premium and higher plans.



https://user-images.githubusercontent.com/77539/137152650-e635f9c6-5926-42e8-96ee-3462294004d1.mov
